### PR TITLE
docs: Update readme instructions

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Update the instructions to deploy the pattern to follow IBM style and word usage and for clarity.

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by vburckhardt** [comment]
Very nice. 👍

**Review by vburckhardt — DISMISSED** [review]

**Comment by SirSpidey** [comment]
@ocofaigh You mentioned that there might be changes needed to the flow of this readme. Before I go further with this, can you (or someone else) review the readme for accuracy?

File: README.md [hunk 1]
```diff
@@ -1,246 +1,223 @@
...
 
-By using this architecture, you can:
+This architecture can help you achieve the following goals:
 
-* Establish Trust: The architecture ensures trust by configuring the IBM Cloud account to align with compliance settings as defined in the [Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about) framework.
-* Ensure Observability: The architecture provides observability by deploying services such as IBM Log Analysis, IBM Monitoring, IBM Activity Tracker, and log retention through Cloud Object Storage buckets.
-* Implement Security: The architecture ensures security by deploying IBM Key Protect and IBM Secrets Manager.
-* Achieve Regulatory Compliance: The architecture ensures regulatory compliance by implementing CI/CD/CC pipelines, along with IBM Security Compliance Center (SCC) for secure application lifecycle management.
+- Establish trust: The architecture configures the IBM Cloud account to align with the compliance settings that are defined in the [IBM Cloud for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about) framework.
```
Comment (ocofaigh) [inline README.md:15]: Hmm, the DA now references the `AI Security Guardrails 2.0` SCC profile, so maybe we should not mention FSCloud here anymore? Thoughts @vburckhardt ?

File: README.md [hunk 2]
```diff
@@ -1,246 +1,223 @@
...
+- Ensure observability: The architecture provides observability by deploying services such as IBM Log Analysis, IBM Monitoring, IBM Activity Tracker, and log retention through IBM Cloud Object Storage buckets.
```
Comment (ocofaigh) [inline README.md:16]: It now also deploy Event Notifications if you want to add that somewhere

File: README.md [hunk 3]
```diff
@@ -1,246 +1,223 @@
...
-* (Recommended to ensure successful sample app deployment) Created or have access to a signing key, which is the base64 key obtained from `gpg --gen-key` without passphrase (if not generated before or expired) and then exported via `gpg --export-secret-key <Email Address> | base64` command. See the [devsecops image signing page](https://cloud.ibm.com/docs/devsecops?topic=devsecops-devsecops-image-signing#cd-devsecops-gpg-export) for details. Keep note of the key for later. The signing key is not required to deploy all of the Cloud resources created by this deployable architecture, but is necessary to get the automation to build and deploy the sample application.
-* (Optional) Installed the IBM Cloud CLI's Project add-on using the `ibmcloud plugin install project` command. More information is available [here](https://cloud.ibm.com/docs/cli?topic=cli-projects-cli).
+    The signing key is not required to deploy all the Cloud resources that are created by this deployable architecture. However, the key is necessary to get the automation to build and deploy the sample application.
+- [Install or update ](https://cloud.ibm.com/docs/cli?topic=cli-getting-started) the IBM Cloud CLI.
```
Comment (ocofaigh) [inline README.md:38]: The CLI isnt actually needed here, and all our steps are for using the UI to deploy the stack, so should we remove this line (and the one under this)?

File: README.md [hunk 4]
```diff
@@ -1,246 +1,223 @@
...
+- Optional: Install the IBM Cloud CLI Project plug-in by running the `ibmcloud plugin install project` command. For more information, see the [Project CLI reference](/docs/cli?topic=cli-projects-cli).
 
-Ensure that you are familiar with the "Important Deployment Considerations" located at the bottom of this document.
+## Add the architecture to a project
```
Comment (ocofaigh) [inline README.md:42]: There is now 2 variations, so we may want to tweek the wording here
image

File: README.md [hunk 5]
```diff
@@ -1,246 +1,223 @@
...
 
 
-## 6. Important Deployment Considerations
-
-### API Key Requirements
+### Enabling Watson Assistant
 
-The deployable architecture can only be deployed with an API Key associated with a user. It is not compatible with API Keys associated with a serviceId. Additionally, it cannot be deployed using the Project trusted profile support.
+After the application is built and is running in Code Engine, enable Watson Assistant in the app.
+To complete the installation, follow the steps that are outlined in the [Configuration steps for watsonx Assistant artifacts](https://github.com/IBM/gen-ai-rag-watsonx-sample-application/blob/main/artifacts/artifacts-README.md) file in the gen-ai-rag-watsonx-sample-application GitHub project.
```
Comment (ocofaigh) [inline README.md:127]: @brendankellyibm is this still necessary?

File: README.md [hunk 6]
```diff
@@ -1,246 +1,223 @@
...
-To undeploy the infrastructure created by the automation, complete the following steps:
+1.  Clean up the configuration
 
-### 1. Delete Resources Created by the CI toolchain
+    This step is optional if you plan to destroy all Watson resources. The artifacts that are created by the application are deleted as part of undeploying the Watson resources.
 
-Those resources are not destroyed automatically as part of undeploying the stack in Project:
-- **Code Engine Project**: Delete the code engine project created for the sample application.
-- **Container Registry Namespace**: Delete the container registry namespace created by the CI tookchain.
+    Follow the steps outlined in the [cleanup.md](https://github.com/IBM/gen-ai-rag-watsonx-sample-application/blob/main/artifacts/artifacts-cleanup.md) file to remove the configuration for the sample app.
```
Comment (ocofaigh) [inline README.md:210]: @brendankellyibm are these steps still needed?

File: README.md [hunk 7]
```diff
@@ -1,246 +1,223 @@
...
 
+1.  Delete resources created by the CI toolchain
 
-### 2. Undeploy Configurations in the Project
+    The following resources, which are created by the toolchain, are not destroyed as part of undeploying the stack in Project.
 
-Select "Undeploy" option in the menu associated with the stack in the project.
-![undeploy](./images/17-undeploy.png)
+    - Code Engine Project
+        - Delete the code engine project created for the sample application.
```
Comment (ocofaigh) [inline README.md:217]: You no longer need to manually do this - the code engine projects are now managed by terraform in the latest version

**Review by ocofaigh — CHANGES_REQUESTED** [review]
I left a few comments - probably need inputs from others too. 
There is now a second variation which hosts the app on roks instead of code engine, so we likely want to update some of the docs here for that. I'll ping you a doc that Anuj was working on locally too

File: README.md (hunk 5 above)
Comment (brendankellyibm) [reply README.md:127]: I don't believe so - none of the Discovery pieces are applicable to Elastic Search installations, and we seem to have the assistant skills in the DA already. @ajainghub might be able to confirm. If so, we need to clean up that doc file too.

File: README.md (hunk 6 above)
Comment (brendankellyibm) [reply README.md:210]: The assistant should be removed with the undeploy of the app config DA, so I don't think this is needed any more.

**Review by vburckhardt — APPROVED** [review]

**Comment by vburckhardt** [comment]
Let's merge this version which applies to what is currently in community catalog. We'll make other updates that align with the next release.

File: README.md (hunk 1 above)
Comment (vburckhardt) [reply README.md:15]: AI Security Guardrails is a superset of FsCloud, so I think we can mention both.

File: README.md (hunk 3 above)
Comment (vburckhardt) [reply README.md:38]: Yes

File: README.md [hunk 8]
```diff
@@ -1,246 +1,223 @@
...
-* (Recommended to ensure successful sample app deployment) Created or have access to a signing key, which is the base64 key obtained from `gpg --gen-key` without passphrase (if not generated before or expired) and then exported via `gpg --export-secret-key <Email Address> | base64` command. See the [devsecops image signing page](https://cloud.ibm.com/docs/devsecops?topic=devsecops-devsecops-image-signing#cd-devsecops-gpg-export) for details. Keep note of the key for later. The signing key is not required to deploy all of the Cloud resources created by this deployable architecture, but is necessary to get the automation to build and deploy the sample application.
-* (Optional) Installed the IBM Cloud CLI's Project add-on using the `ibmcloud plugin install project` command. More information is available [here](https://cloud.ibm.com/docs/cli?topic=cli-projects-cli).
+    The signing key is not required to deploy all the Cloud resources that are created by this deployable architecture. However, the key is necessary to get the automation to build and deploy the sample application.
```
Comment (rajatagarwal-ibm) [inline README.md:37]: I did some testing as per https://github.ibm.com/GoldenEye/issues/issues/9213 issue... my findings are as follows:

Without the singing key, the sample app gets deployed successfully and after that it triggers the CI pipeline. In the CI pipeline the step `build-sign-artifact` failed due to not passing signing key but overall the CI pipeline passes anyway. 

@SirSpidey would you be able to update the text based on these findings?

Comment (SirSpidey) [inline README.md:37]:
```suggestion
    The signing key is not required to deploy the IBM Cloud resources that are created by this deployable architecture to Code Engine. However, without the key, the CI pipeline fails.
```

@rajatagarwal-ibm Thoughts?

Comment (vburckhardt) [reply README.md:37]: Thanks - I'll just add that the application itself deploys, but the pipeline is marked as failed to indicate that the signing step did not complete successfully.

**Review by vburckhardt — APPROVED** [review]

**Pull request merged**